packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -135,38 +142,32 @@ const IDVForm = ({
-                                                                {...field}
-                                                                name='document_type'
-                                                                data-lpignore='true'
-                                                                error={touched.document_type && errors.document_type}
-                                                                autoComplete='off'
-                                                                type='text'
-                                                                label={localize('Choose the document type')}
-                                                                list_items={document_list}
-                                                                value={values.document_type.text}
-                                                                onBlur={(e: { target: HTMLInputElement }) => {
-                                                                    handleBlur(e);
-                                                                    if (!getDocument(e.target.value)) {
-                                                                        resetDocumentItemSelected();
-                                                                    }
-                                                                }}
-                                                                onChange={handleChange}
-                                                                onItemSelection={(item: TDocument) => {
-                                                                    if (
-                                                                        item.text === 'No results found' ||
-                                                                        !item.text
-                                                                    ) {
-                                                                        setSelectedDoc('');
-                                                                        resetDocumentItemSelected();
-                                                                    } else {
-                                                                        bindDocumentData(item);
-                                                                    }
-                                                                }}
-                                                                required
-                                                            />
-                                                        </div>
+                                                        <Autocomplete
+                                                            {...field}
+                                                            data-lpignore='true'
+                                                            error={touched.document_type && errors.document_type}
```

Comment:
There's a type error for `TDocument` is not assignable, can you please check on this?

Reply:
I'll work on the TS fixes as a tech debt card.